app/javascript/app/webauthn.ts

```diff
@@ -89,25 +110,32 @@ const enrollWebauthnDevice = ({
       ],
       timeout: 800000,
       attestation: 'none',
-      excludeList: [],
```

Comment:
This appears to have been an option in an early draft that was never standardized.

Draft: https://www.w3.org/TR/2017/WD-webauthn-20170505/#dictionary-makecredentialoptions
Final: https://www.w3.org/TR/webauthn-1/ (no mention)